spec/lib/sarif/osv/maven_osv_sarif_spec.rb

```diff
@@ -0,0 +1,137 @@
+require_relative '../../../spec_helper'
+require 'json'
+
+describe Sarif::MavenOSVSarif do
```

Comment:
Please call `validate_sarif` on the result sarifs. You can grep for it to see how it's used in other specs.